src/handlers/key.go

```diff
@@ -76,7 +76,7 @@ func GetKey(ctx context.Context, settings *kmipapi.ConfigurationSettings, line s
 		return
 	}
 
-	fmt.Printf("get key for uid (%s) key is (%v)\n", uid, key)
+	fmt.Printf("get key for uid (%s) key is (%v)\n", uid, *key)
```

Comment:
We should not log the entire key value since this a security risk. I suggest you create a function such as `SecureKeyPrint()` that can be used in all locations to only print the last `X` characters of the key (without any dashes). For example, last4  := *key[len(*key)-4:]. If we do this, and discover 4 is not enough, then we could change it to 6 in a single location.